clients/src/main/resources/common/message/SaslAuthenticateResponse.json

```diff
@@ -29,6 +29,6 @@
     { "name": "AuthBytes", "type": "bytes", "versions": "0+",
       "about": "The SASL authentication bytes from the server, as defined by the SASL mechanism." },
     { "name": "SessionLifetimeMs", "type": "int64", "versions": "1+", "default": "0", "ignorable": true,
-      "about": "The SASL authentication bytes from the server, as defined by the SASL mechanism." }
+      "about": "The duration in milliseconds that a successful session response is valid for." }
```

Comment:
Maybe we can reuse the original message?
https://github.com/apache/kafka/blob/2.2/clients/src/main/java/org/apache/kafka/common/requests/SaslAuthenticateResponse.java#L51

Reply:
Ah cool, thanks for providing context/history, I'll update this in the next few days.

Reply:
I updated with original copy as suggested.